tests/perforce_utils.py

```diff
@@ -194,6 +194,21 @@ def perforce_workspace(request, perforce_connection, tmpdir):
         change["Description"] = "Test submit"
         p4.run_submit(change)
 
+        permissions = p4.fetch_protect()
+        permissions['Protections'] = [
+            'write user * * //...',
+            'list user * * -//spec/...',
+            'super user p4user * //...',
+            '=write user p4user * -//depot/write-protected/...',
+            '=open user p4user * -//depot/open-protected/...'
+        ]
+        p4.save_protect(permissions)
+
+        triggers = {'Triggers': [
+            'test.check change-submit //depot/review-protected/... "/bin/cp . ."'
```

Comment:
The intention of using "/bin/cp . ." is not clear

Reply:
This will try to launch any executable on server side. This is just an executable granted to exist and parameters granted to fail. The alternative is to commit an always-failing script to repo, but I didn't think it was actually needed.

Reply:
The more clear way is to use `false` command